README.md

```diff
@@ -78,10 +78,88 @@ This project is built with specific integrations to enhance its functionality an
+
+#### Decap Configuration
+
+Decap's setup includes two configuration files:
+
+1. `public/admin/config.yml` - Primarily contains the schema and metadata for the content, but also includes authentication settings among other things,
+2. `public/admin/index.html` - The HTML template to view and edit the content in a web interface. Access it via [/admin/index.html](http://localhost:3000/admin/index.html#/)
+
+Additionally, `src/app/_data/cmsConfigSchema.json` is a file autogenerated when the server starts, with `predev` and `prebuild` commands. It does not require manual editing and should probably be part of `.gitignore` in the future.
+
+#### Markdown files
+
+We use [markdown](https://www.markdownguide.org/cheat-sheet/) files for the content of our websites. It allows us to convert human-readable text into HTML. Read more about [markdown](https://www.markdownguide.org/cheat-sheet/) here:
+
+Each markdown file typically consists of two parts:
+
+1. The header section: Enclosed within `--` tags, this YAML-formatted section contains metadata about the file, similar to what you would find in HTML `<head>` tags: title, description, image links, etc.
+2. The body section: Contains the content itself in markdown format.
+
+Here’s an example:
+
+```markdown
+---
+title: Hello
+slug: home
+---
+
+# Hello world!
+
+lorem ipsum
+```
+
+Metadata is processed using the [gray-matter](https://www.npmjs.com/package/gray-matter) library, transforming it into an object usable within our code to display content in the relevant places:
+
+```js
+{
+  data: { title: 'Hello', slug: 'home' }
+  content: '<h1>Hello world!</h1><p>lorem ipsum</p>',
+}
+```
+
+#### How to Update Content
+
+Two scenarios
+
+**Updating Existing Content**
+
+If you only need to update existing content, no need to update Decap’s configuration. Modify the markdown file for the content you wish to update. For example, to change the title on the home page:
+
+1. Open `src/content/pages/home.md`.
+2. Edit the `title` in the header section.
+
+The website should hot reload and changes should appear immediately in your local development environment. Once you’re happy with the update, commit your changes via git once and voila
+
+**Adding or Removing Content**
+
+If you need to add or remove content from a page or a section, like a new subtitle on the home page, then Decap must know about it. Hence, you need to update both the markdown and the configuration file.
+
+1. Add the new content in the markdown file, e.g., a `subtitle` in the header of `src/content/pages/home.md`.
+2. Update `public/admin/config.yml` to reflect this addition under the appropriate section, for example:
+
+_If you only do step 1, you will still be able to see the change in your code, but it will not be reflected in the Decap dashboard for the content team._
```

Comment:
What I'm trying to say is that if a developer adds content to a markdown file without reflecting these changes to `config.yml`, it will still work for them. They can `console.log(attributes)` in their code and will see the new field there, but it will not appear in the Decap dashboard.

This is to draw attention to how Decap works and emphasize that we are using it for collaboration between the dev and content teams.

Reply:
I think we can remove it.

```suggestion
```